cordova/package.json

```diff
@@ -11,49 +11,50 @@
     "plugins": {
       "cordova-plugin-secure-storage": {},
       "cordova-plugin-dialogs": {},
-      "cordova-plugin-network-information": {},
-      "phonegap-plugin-barcodescanner": {
-        "ANDROID_SUPPORT_V4_VERSION": "27.+"
-      },
       "cordova-plugin-splashscreen": {},
       "cordova-clipboard": {},
       "cordova-plugin-device": {},
       "cordova-plugin-safariviewcontroller": {},
-      "cordova-plugin-fingerprint-aio": {
-        "FACEID_USAGE_DESCRIPTION": "Unlock Solar…"
-      },
       "cordova-plugin-customurlscheme": {
         "URL_SCHEME": "web+stellar",
         "ANDROID_SCHEME": " ",
         "ANDROID_HOST": " ",
         "ANDROID_PATHPREFIX": "/"
       },
       "cordova-plugin-ionic-keyboard": {},
-      "cordova-plugin-local-notification": {},
-      "cordova-plugin-androidx-adapter": {}
+      "cordova-plugin-androidx-adapter": {},
+      "@red-mobile/cordova-plugin-barcodescanner": {
+        "ANDROIDX_LEGACY_SUPPORT_V4_VERSION": "1.0.0"
+      },
+      "cordova-plugin-fingerprint-aio": {
+        "FACEID_USAGE_DESCRIPTION": "Login now...."
+      },
+      "cordova-plugin-network-information": {},
+      "cordova-plugin-proguard": {}
     }
   },
   "dependencies": {
-    "cordova": "^11.0.0",
+    "cordova": "^12.0.0",
     "cordova-clipboard": "^1.2.1",
     "cordova-plugin-badge": "^0.8.8",
-    "cordova-plugin-device": "^2.0.2",
     "cordova-plugin-dialogs": "^2.0.1",
-    "cordova-plugin-fingerprint-aio": "^3.0.1",
     "cordova-plugin-ionic-keyboard": "^2.2.0",
-    "cordova-plugin-local-notification": "^0.9.0-beta.2",
-    "cordova-plugin-network-information": "^2.0.1",
     "cordova-plugin-safariviewcontroller": "^2.0.0",
-    "cordova-plugin-splashscreen": "^6.0.0",
-    "phonegap-plugin-barcodescanner": "^8.0.1"
+    "cordova-plugin-splashscreen": "^6.0.0"
   },
   "devDependencies": {
+    "@moodlehq/cordova-plugin-local-notification": "^0.9.0-moodle.11",
```

Comment:
was it moved to devDependencies on purpose?

Reply:
No, this file is operated by Cordova CLI. What I did was "cordova plugin remove <plugin>" and "cordova plugin add <plugin>" with a couple of plugins. It's better to trust Cordova, I think.